paper/paper.md

```diff
@@ -92,8 +89,8 @@ increases drastically with an increasing number of notations.
 Consequentially, for even a small number of different notations, the implementation of all corresponding translation methods is not feasible.
 Therefore, `mechkit` does not necessarily directly translate one notation into another.
 Instead, in the case of the translation of second- and fourth-order tensors,
-`mechkit` determines the shortest path between source and target notation as illustrated in
-the graph of supported notations, see \autoref{fig:stiffness_graph}.
+`mechkit` determines the shortest path between source and target in the set of implemented notations as illustrated in
```

Comment:
This is a really neat feature and we should consider to emphasize this even further.